builders/build/historical-updates.md

```diff
@@ -335,6 +335,24 @@ You can review the [relative Frontier PR](https://github.com/polkadot-evm/fronti
 
 ---
 
+#### EVM Tracing Weight Limits {: #evm-tracing-weight-limits }
```

Comment:
What about EVM tracing weight limits that makes this a bug? I would focus it more on the actual problem. Maybe something like "Skipped Ethereum Transaction Traces"

Reply:
For sure!